syntax="proto3";


message OpDef{
    string name=1;
}
